lib/kb_maxbin/Utils/MaxBinUtil.py

```diff
@@ -405,6 +405,7 @@ def run_maxbin(self, params):
         log('Saved result files to: {}'.format(result_directory))
         log('Generated files:\n{}'.format('\n'.join(os.listdir(result_directory))))
 
+        binned_contig_obj_ref = None
```

Comment:
unnecessary initialization

Reply:
initialization to None is never bad and not a waste of code